pkg/apis/apis.go

```diff
@@ -19,13 +19,15 @@ package apis
 import (
 	"k8s.io/apimachinery/pkg/runtime"
 
-	"sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	clusterv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/karpenter-provider-cluster-api/pkg/apis/v1alpha1"
```

Comment:
and call this `api`, also to keep consistent with other usages

Reply:
done

Reply:
it's a good point, i was only using `api` during this phase of development, i think we should come up with a better name and refactor the code to use it.